packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service.ts

```diff
@@ -5,6 +5,7 @@ import { ApplicationLoadBalancer, ApplicationProtocol, NetworkLoadBalancer } fro
 import * as iam from '@aws-cdk/aws-iam';
 import * as cdk from '@aws-cdk/core';
 import { Test } from 'nodeunit';
+import {FargatePlatformVersion} from '../../../aws-ecs/lib/fargate/fargate-service';
```

Comment:
Since you're already importing the entire ecs module, you can refer to it with ecs.FargatePlatformVersion